content/en/serverless/azure_functions/_index.md

```diff
@@ -0,0 +1,125 @@
+
+2. **Start the Datadog serverless compatibility layer and initialize the Node.js tracer**. Add the following lines to your main application entry point file (for example, `app.js`):
+
+   ```js
+   require('@datadog/serverless-compat').start();
+
+   // This line must come before importing any instrumented module. 
+   const tracer = require('dd-trace').init()
+   ```
+
+   [1]: https://github.com/DataDog/dd-trace-js?tab=readme-ov-file#version-release-lines-and-maintenance
+   [2]: /tracing/trace_collection/automatic_instrumentation/?tab=singlestepinstrumentation
+{{< /programming-lang >}}
+{{< programming-lang lang="python" >}}
+1. **Install dependencies**. Run the following commands:
+   ```shell
+   pip install datadog-serverless-compat
+   pip install ddtrace
+   ```
+
+   You must use `ddtrace` v2.19+.
+
+   Datadog recommends using the latest versions of both `datadog-serverless-compat` and `ddtrace` to ensure you have access to enhancements and bug fixes.
+
+2. **Initialize the Datadog Python tracer and serverless compatibility layer**. Add the following lines to your main application entry point file:
+
+   ```python
+   from datadog_serverless_compat import start
+   from ddtrace import tracer, patch_all
+
+   start()
+   patch_all()
+   ```
+
+{{< /programming-lang >}}
+{{< /programming-lang-wrapper >}}
+
+3. **Deploy your function**.
+
+4. **Configure Datadog intake**. Add the following environment variables to your function's application settings:
+   ```
+   DD_API_KEY=<YOUR_API_KEY>
+   DD_SITE={{< region-param key=dd_site code="true" >}}
+   ```
+   - Set `DD_API_KEY` to your [Datadog API key][1].
+   - Set `DD_SITE` to {{< region-param key=dd_site code="true" >}}. (Ensure that your correct [Datadog site][2] is selected in the drop-down on the right side of this page.)
+
+5. **Correlate traces with metrics by setting tags**. You can collect metrics from your Azure Functions by installing the [Datadog Azure integration][6]. To correlate these metrics with your traces, configure the following tags as environment variables. You can add custom tags as `DD_TAGS`.
+
+   ```
+   DD_ENV="<ENVIRONMENT>"
+   DD_SERVICE="<SERVICE_NAME>"
+   DD_VERSION="<VERSION>"
+   DD_TAGS="<KEY_1:VALUE_1>,<KEY_2:VALUE_2>"
+   ```
+
+## View traces in Datadog
+
+In Datadog, navigate to the [Trace Explorer][4] page and search for the service name you set in the `DD_SERVICE` environment variable
+
+<!-- TODO: maybe some more here -->
+
+You can also view traces on the [Serverless > Azure Functions][5] page. This view is enriched with telemetry collected by the [Datadog Azure integration][6].
+
+## Enable debug logs
+
+You can collect debug logs for troubleshooting. To configure debug logs, use the following environment variables:
+
+`DD_TRACE_DEBUG`
+: Enables (`true`) or disables (`false`) debug logging for the Datadog Tracing Library. Defaults to `false`.
+
+  **Values**: `true`, `false` 
```

Comment:
What about linking to this page for tracer debug logs?
https://docs.datadoghq.com/tracing/troubleshooting/tracer_debug_logs/#enable-debug-mode